db/migrate/20230503123537_create_comments.rb

```diff
@@ -0,0 +1,13 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.date :createdat
+      t.date :updatedat
```

Comment:
- [optional] The createdat and updatedat columns are automatically created by rails  using the `t.timestamps`. Kindly remove these please.